<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerView</class>
 <widget class="QDialog" name="customerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Customer</string>
  </property>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>号顾客：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>欢迎使用餐桌预定系统！</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>现在餐桌情况如下：</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>661</width>
     <height>121</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>小型桌子</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>中型桌子</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>大型桌子</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>能容人数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>可用桌数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>下一顾客</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>排队人数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>直接入座</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>加入排队</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="NumLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>您的排号为：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Num">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目前还没加入排队！</string>
   </property>
  </widget>
  <widget class="QLabel" name="FrontLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>在您前面还有：</string>
   </property>
  </widget>
  <widget class="QLabel" name="People">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="cL2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>位在等待</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Small">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Medium">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Large">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>大型</string>
   </property>
  </widget>
  <widget class="QLabel" name="ID">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit1Button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="smallIn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小型座</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mediumIn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中型座</string>
   </property>
  </widget>
  <widget class="QPushButton" name="largeIn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>大型座</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>您的桌号为：</string>
   </property>
  </widget>
  <widget class="QLabel" name="TableNum">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目前还没有桌号！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Exit1Button</sender>
   <signal>clicked()</signal>
   <receiver>customerView</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
